rules/windows/process_creation/proc_creation_win_susp_regsvr32_no_dll.yml

```diff
@@ -18,18 +18,17 @@ detection:
     selection:
         Image|endswith: '\regsvr32.exe'
     filter:
-        CommandLine|contains: 
+        - CommandLine|contains:
             - '.dll'
             - '.ocx'
             - '.cpl'
             - '.ax'
             - '.bav'
             - '.ppl'
-    filter_null1_for_4688:
-        CommandLine: null
-    filter_null2_for_4688:
-        CommandLine: ''
-    condition: selection and not filter and not filter_null1_for_4688 and not filter_null2_for_4688
+        - CommandLine:
+            - null
```

Comment:
🃏 If I remenber some backend breack the detection with `null` in a list.
Need to be check , `null` is a special case .

Reply:
Yes, correct. A selection with "null" always has to be separate. It cannot be combined with string expressions.